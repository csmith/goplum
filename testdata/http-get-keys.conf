alert debug.sysout "test" {}

check http.get "test-get" {
  url = "https://www.example.com/"
  content = "Example Domain"
  content_expected = false
  certificate_validity = 10d
  min_status_code = 400
  max_status_code = 499
  auth {
    username = "acidburn"
    password = "HackThePlanet"
  }
}