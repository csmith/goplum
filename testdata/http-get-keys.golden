{
  "Alerts": {
    "test": {}
  },
  "Checks": {
    "test-get": {
      "Name": "test-get",
      "Type": "http.get",
      "Config": {
        "Alerts": [
          "*"
        ],
        "Groups": [],
        "Interval": 30000000000,
        "Timeout": 20000000000,
        "GoodThreshold": 2,
        "FailingThreshold": 2
      },
      "Check": {
        "Url": "https://www.example.com/",
        "Auth": {
          "Username": "acidburn",
          "Password": "HackThePlanet"
        },
        "Content": "Example Domain",
        "ContentExpected": false,
        "CertificateValidity": 864000000000000,
        "MinStatusCode": 400,
        "MaxStatusCode": 499
      },
      "LastRun": "0001-01-01T00:00:00Z",
      "Scheduled": false,
      "Settled": false,
      "State": "indeterminate",
      "Suspended": false,
      "History": [
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null
      ]
    }
  },
  "Groups": {}
}